weed/iamapi/iamapi_test.go

```diff
@@ -152,6 +153,53 @@ func TestPutUserPolicy(t *testing.T) {
 	assert.Equal(t, http.StatusOK, response.Code)
 }
 
+func TestPutUserPolicyError(t *testing.T) {
```

Comment:
I added this test to verify that we can actually return a valid error code and message.